.circleci/config.yml

```diff
@@ -3,9 +3,10 @@ version: 2.1
 orbs:
     slack: circleci/slack@4.4.4
     gravitee: gravitee-io/gravitee@1.0.44
-    secrethub: secrethub/cli@1.1.0
+    #secrethub: secrethub/cli@1.1.0
     gh: circleci/github-cli@1.0.5
     aws-cli: circleci/aws-cli@2.0.6
+    apim: gravitee-io/gravitee-apim@1.0
```

Comment:
🟥  Nop we don't want to have this orb back as we did a lot of effort to clean the mess that was there.

Also, see all my feedbacks on the orb PR: https://github.com/gravitee-io/gravitee-apim-cci-orb/pull/13